src/components/Insight/ProfileInfo.js

```diff
@@ -9,20 +9,25 @@ const ProfileInfo = ({
   networkStatus,
   picUrl,
   className = '',
-  infoClassName = ''
+  infoClassName = '',
+  noPic
 }) => {
   return (
     <div className={cx(styles.wrapper, className)}>
-      <div className={styles.icon}>
-        {picUrl ? (
-          <img src={picUrl} alt='Profile Pic' />
-        ) : (
-          <Icon type='profile-round' />
-        )}
-        {networkStatus && (
-          <div className={cx(styles.onlineIndicator, styles[networkStatus])} />
-        )}
-      </div>
+      {!noPic && (
```

Comment:
@DmitriVanGuard Logic here is "no" + "no" + Pic && (make smth). Condition starts with negative is less readable.
Possible better make withPic. Default value is true. And condition withPic &&.

Also, we can move all code after && () to a variable before return. Or we can remove/reduce "return".